exampleCourse/questions/template/multiple-choice/numeric/question.html

```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+  <p>If $a = {{params.a}}$ and $b = {{params.b}}$, then what is $a$ multiplied by $b$?</p>
+</pl-question-panel>
+
+<pl-multiple-choice answers-name="ans">
+  <pl-answer correct="true">${{params.correct_answer}}$</pl-answer>
+  <pl-answer correct="false">${{params.wrong_answer1}}$</pl-answer>
```

Comment:
Nit: Put wrong answers into a list and iterate through. That's a more extendable pattern if people use this in their own questions.

Reply:
Heh, great minds: https://github.com/PrairieLearn/PrairieLearn/pull/9407#discussion_r1503186233

Reply:
I buy the argument from @mwest1066 that we actually want to avoid looping for example questions. I'm going to leave this as is.